src/frontend/efiling-frontend/package.json

```diff
@@ -99,7 +99,7 @@
     "collectCoverageFrom": [
       "src/**/*.js",
       "!src/*.js",
-      "!src/components/hoc/*.js",
+      "!src/domain/authentication/*.js",
```

Comment:
where the test ignore for auth before ?

Reply:
Correct, there wasn't any tests for the auth before.  Rather than make this PR too large, we can add a test later.